src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -964,25 +963,25 @@ public static VariantContext trimAlleles(final VariantContext inputVC, final boo
             return inputVC;
 
         // see whether we need to trim common reference base from all alleles
-        final int revTrim = trimReverse ? computeReverseClipping(inputVC.getAlleles(), inputVC.getReference().getDisplayString().getBytes()) : 0;
-        final VariantContext revTrimVC = trimAlleles(inputVC, -1, revTrim);
-        final int fwdTrim = trimForward ? computeForwardClipping(revTrimVC.getAlleles()) : -1;
+        final int revTrim = trimReverse ? computeClippingLength(inputVC.getAlleles(),true) : 0;
+        final VariantContext revTrimVC = trimAlleles(inputVC, 0, revTrim);
+        final int fwdTrim = trimForward ? computeClippingLength(revTrimVC.getAlleles(),false) : 0;
         return trimAlleles(revTrimVC, fwdTrim, 0);
     }
 
     /**
-     * Trim up alleles in inputVC, cutting out all bases up to fwdTrimEnd inclusive and
+     * Trim up alleles in inputVC, cutting out all bases up to fwdTrim inclusive and
      * the last revTrim bases from the end
      *
      * @param inputVC a non-null input VC
-     * @param fwdTrimEnd bases up to this index (can be -1) will be removed from the start of all alleles
+     * @param fwdTrim bases up to this index (can be -1) will be removed from the start of all alleles
```

Comment:
This is now the number of bases to be removed, not the index up to which you can remove.